controllers/invitation.go

```diff
@@ -23,7 +25,7 @@ var invitation *template.Template
 
 func init() {
 	var err error
-	invitation, err = template.ParseFiles("./mail/template.invitation")
+	invitation, err = template.ParseFiles(path.Join(os.Getenv("GOPATH"), "src/github.com/coduno/api/mail/template.invitation"))
```

Comment:
Well, there are various solutions to this, but depending on `$GOPATH` clearly is hacky. Please decouple and make ``controllers/sub.go` not rely on a path, but rather make that "configurable". For example, it would make sense to move the responsibility of loading the templates (or only specifying the path) to `main`. The tests could then use their own templates, by configuring the handler differently. Does that sound reasonable?